.pre-commit.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env sh
+# pre-commit.sh
+
+STASH_NAME="pre-commit-$(date +%s)"
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m'
+
+if git stash save -u -k -q $STASH_NAME; then
+    echo "${YELLOW}Stashed changes as:${NC} ${STASH_NAME}\n\n"
+    stash=1
```

Comment:
@vitvakatu Why that? Without setting the `stash` flag the changes aren't applied afterwards.